python_src/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '1.3'
+__version__ = '1.42'
```

Comment:
Probably the new version should be `1.4`, and for now it's best to make it a beta version e.g. `1.4b1`.